src/Gui/Selection/Selection.h

```diff
@@ -48,7 +48,7 @@ namespace App
 namespace Gui
 {
 
-enum class ResolveMode {

```

Comment:
I don't find this renaming an improvement. It's only making the name more inconvenient, and ResolveMode was never difficult knowing it being related to selection.